### PR TITLE
Add Ishkar's Felshield Emitter to Items

### PR DESCRIPTION
**Pull request by hartra344**

### CODE REVIEW

File: src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        const spellId = event.ability.guid;
+        if (spellId === SPELLS.FELSHIELD_ABSORB.id) {
+            this.damageAbsorbed += event.amount;
+        }
+    }
+
+    on_byPlayer_damage(event) {
+      const spellId = event.ability.guid;
+      if (spellId === SPELLS.FELSHIELD_DAMAGE.id) {
+        this.damageDealt += event.amount;
```
Comment (MartijnHols) [inline src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js]: To be consistent with the rest of the code base please also include `event.absorbed`. Absorbed damage usually is beneficial to the raid.

**Comment by MartijnHols** [comment]
Thanks for the PR, a minor thing before we can merge this ;)

File: src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js (hunk 1 above)
Comment (hartra344) [reply src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js]: I'm a bit confused though by what you're saying. The item's description is: 
Place a Felshield on an ally, absorbing X damage for 9 sec. When the shield is consumed or expires, it explodes dealing 50% of the absorbed damage as Fire split amongst all enemies within 8 yds. (1 Min Cooldown)

This means it has 2 effects, absorb and then blow up to damage enemies for half the amount absorbed. This specific part is the damage to enemies part, not the absorbed part.

Comment (Pewtro) [reply src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js]: ``this.damageDealt += event.amount + (event.absorbed || 0);``

Comment (hartra344) [reply src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js]: OH! Nevermind, you mean how much the boss absorbed. got it!

**Comment by MartijnHols** [comment]
Ohh one more thing (I always forget this); please add a changelog entry to the core changelog :D

**Pull request merged**